CorsixTH/Lua/game_ui.lua

```diff
@@ -792,18 +785,23 @@ function GameUI:playAnnouncement(name, priority, played_callback, played_callbac
   self.announcer:playAnnouncement(name, priority, played_callback, played_callback_delay)
 end
 
-function GameUI:onTick()
-  local repaint = UI.onTick(self)
-
-  -- get the mouse button for dragging the map
-  local mouse_drag_button = nil
+-- Get the configured mouse drag button
+function GameUI:_isMouseDragButtonDown()
```

Comment:
The function name suggests you obtain a boolean value whether the mouse drag button is down or not.
The documentation says you are getting the configured mouse drag button.

Please make both things say the same.

The idea of documentation is that it briefly describes what the function is doing so you don't have to dig in the code to reverse engineer that information. For this function it's not so bad, but if you have 300 line function (which happens quickly in lua), reading say 3 lines of text and the input/output relation is much quicker than reading 300 lines of details and then hopefully correctly deduce the higher level aim of the intention of the function.